src/applications/gi/components/profile/InstitutionProfile.jsx

```diff
@@ -43,13 +43,13 @@ export class InstitutionProfile extends React.Component {
       gibctEstimateYourBenefits,
     } = this.props;
     return (
-      <div>
+      <div className="usa-accordion">
         <HeadingSummary
           institution={profile.attributes}
           onLearnMore={showModal.bind(this, 'gibillstudents')}
           onViewWarnings={this.handleViewWarnings}
         />
-        <div className="usa-accordion">
+        <div className="usa-accordion vads-u-margin-top--1p5 eybcontainer">
```

Comment:
this is not the Estimated your benefits container, that is in `src/applications/gi/containers/EstimateYourBenefits.jsx` this is the `div` that includes all of the major accordions and should have the `usa-accordion` class

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/containers/EstimateYourBenefits.jsx#L90

Reply:
Gotcha, that is a good point. I just needed a unique classname for this to apply css to it so the additional top spacing would appear at page widths above 641px. I will change the name to something more appropriate.

Reply:
Had some hesitancy to even call that a container in the first place, since it is not a container from the react.js perspective, more so a container of content.

Reply:
is this for AC 9 and 10? if so that work is being covered in https://github.com/department-of-veterans-affairs/vets-website/pull/12687